src/main/java/model/data/DataTableBuilder.java

```diff
@@ -16,13 +16,15 @@
 public class DataTableBuilder {
 	private List<DataRow> rows;
 	private List<DataColumn> columns;
+	private String name;
 
 	/**
 	 * Create a new builder.
 	 */
-	public DataTableBuilder() {
-		 rows =  new ArrayList<DataRow>();
-		 columns = new ArrayList<DataColumn>();
+	public DataTableBuilder(String name) {
```

Comment:
Het is een beetje vaag om in een builder waarbij je alles doet, nadat je hem aangemaakt hebt, wel de naam in de constructor mee te geven.